keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/probabilistic/GaussianVertex.java

```diff
@@ -126,4 +133,19 @@ public PartialDerivatives forwardModeAutoDifferentiation(Map<Vertex, PartialDeri
             return PartialDerivatives.withRespectToSelf(this.getId(), this.getShape());
         }
     }
+
+    @Override
+    public List<KeanuSavedBayesNet.NamedParent> getNamedParents() {
```

Comment:
It feels wrong that `getParents` and `getNamedParents` are logically independent of each other.

Reply:
Ok - I'll try and think of a better name for this.